.dockerignore

```
@@ -1,2 +1,9 @@
+assets/
```

I'm also curious if we need `app/apptesting` but that would require testing

```suggestion
assets/
build/
```